src/DynamoCoreWpf/UI/Themes/Modern/DynamoModern.xaml

```diff
@@ -5352,4 +5344,104 @@
             </Trigger>
         </Style.Triggers>
     </Style>
+    
```

Comment:
Moving the expander style here from the slider component so we can use it in the dropdown as well.